### PR TITLE
[Ready] Item to cargo for R&D engineering tech

### PR DESCRIPTION
**Pull request by LatD**
Currently there is only one way of getting engineering tech and cargo has very few ways of being useful to R&D so this helps with both of those. This adds an item which does nothing but give an alternative way of getting high engineering tech in R&D.

Sprite from IRC from basil:
machineprototype

:cl: Lati
rscadd: Nanotrasen has added one of their rare machine prototypes to cargo's selection. It might be valuable to research.
/:cl:

### CODE REVIEW

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -800,6 +799,14 @@
 	crate_type = /obj/structure/closet/crate/secure
 	dangerous = TRUE
 
+/datum/supply_pack/science/research
+	name = "Machine Prototype Crate"
+	cost = 5000
```
Comment (ChangelingRain) [inline code/modules/cargo/packs.dm]: Could probably stand to be much higher.

Comment (LatD) [inline code/modules/cargo/packs.dm]: I've never played cargo so I don't know about the prices. I just gave it something that looked pretty high-ish when comparing to other items. Better suggestions for the cost?

Comment (Dorsisdwarf) [inline code/modules/cargo/packs.dm]: 5000 = 50 old points, I think. Pretty sure cargo can buy this just with the cash from selling starting crates

**Comment by Core0verload** [comment]
>This adds an item which does nothing but give an alternative way of getting high engineering tech in R&D.

Useless items are _useless_. Give high engineering to something like RCD. You can get RCDs from both engineering and cargo.

**Comment by LatD** [comment]
You can also make RCDs in a few minutes after round start without ever leaving R&D room. No.

**Comment by optimumtact** [comment]
Have you asked in the spriting subforum for a sprite?

**Comment by LatD** [comment]
I've asked in IRC and I should be getting one from there, hopefully today.

**Comment by Core0verload** [comment]
The idea behind this is so defective it hurts. How about you add engineering to something that's worth it instead of packing it into a useless 5000cr item?

**Comment by LatD** [comment]
I haven't found any perfect items that would fit into engineering tech or magnets tech. I like the point of this since it allows R&D to add some useful item to cargo behind this level gate which will be the next part.

**Comment by LatD** [comment]
It has a sprite now. Do I need to set any icon_states or item_states or is this ready like this? I haven't uploaded sprites before.

**Comment by LatD** [comment]
Slow internet and using a laptop is causing me to make a lot of useless commits, sorry about those.

Final thing: do I need to define an item_state or is this fine like this? I don't even know how the item_state works since I only have one sprite.

**Comment by optimumtact** [comment]
@tgstation/commit-access

**Comment by Core0verload** [comment]
Before, it was a shitty idea. Now it's a shitty idea with a shitty sprite.

File: code/modules/cargo/packs.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/cargo/packs.dm]: ^

**Comment by Jordie0608** [comment]
I'm inclined to agree that this isn't a wonderful idea. It's essentially a snowflake item with a cheap price that does nothing but make RD even easier.

**Comment by LatD** [comment]
Currently the only way to get high engineering tech is to use genetics DNA syringes so this is a much needed alternative to it. Items behind these high engineering tech levels are all items that used to be very easy to get with the old system, such as NV HUDs, jackhammer and thruster implant.

I can tune the cargo cost easily though since I don't know at all what would be a good price. 7000? 8000? Even more? It's supposed to be a bit expensive but still not hard to get if someone is actually working with the stocks or something.

**Comment by Core0verload** [comment]
Give it to some of the crap found in engineering and cargo. RCD/RPD are good options. If scientists would start building their own lathes to make RCDs/RPDs, just raise lathe board required tech.

**Comment by Core0verload** [comment]
QUALITY CONTROL

**Comment by HankMoodyExperience** [comment]
:salt:

**Comment by optimumtact** [comment]
@Core0verload feel free to implement your suggestion if you feel so strongly about it.

**Comment by Core0verload** [comment]
I just look at the sprite and feel an urge to vomit. Even if this idea is good, and it isn't, sprite alone should stop this from getting merged.

**Comment by optimumtact** [comment]
Feel free to also provide a better sprite, we love people improving on others work :+1:

**Comment by Core0verload** [comment]
Merging shit and saying "oh you don't like it just improve it" is not a good way to keep your game shit-free.

**Comment by optimumtact** [comment]
So you don't actually care enough to fix it, which implies the item and feature are in fact perfectly adequate and you are complaining because you don't like the concept.

Sorry that not everything conforms to your singular worldview.

**Comment by Core0verload** [comment]
I can implement my suggestion, but chances of replacing this with it are low. Removing is hard today because of "improve not remove", and that's why pre-merge QC is important.

**Comment by Dorsisdwarf** [comment]
I have to agree that merging controversial stuff and saying "Make a revert/update PR to show you care" is very counter-intuitive to the "Do not make reverts unless its really really awful broken bad" general line cherryman gives us

**Comment by optimumtact** [comment]
2 people disagreeing does not make something controversial

**Comment by Dorsisdwarf** [comment]
3 against, 1 for, where the 1 is the PR creator sounds fairly unpopular to me

**Comment by KorPhaeron** [comment]
We don't base our development on whichever 4 random people happened to view the pr at the rime and hit an emoji

**Comment by KorPhaeron** [comment]
And for the record I don't even think this PR is a great idea, but since LatD is overhauling RD it makes sense to let him complete his goals there so we can jusge it as a whole. I'd rather have a cohesive set of ideas random voting from random people.

**Comment by Dorsisdwarf** [comment]
Sure, I guess

**Comment by Iamgoofball** [comment]
@KorPhaeron ive had PR closes by maintainers based on emoji count before

**Pull request merged**